dork/types.py

```diff
@@ -866,7 +896,7 @@ def update(maze):
         plt.axis("equal")
         plt.axis("off")
         plt.draw()
-        # plt.show()
+        plt.show()
```

Comment:
This code shouldn't be here. `update` should not call `plt.show()` as that will open a new figure.